boards.txt

```diff
@@ -6778,6 +6778,189 @@ adafruit_feather_esp32_v2.menu.DebugLevel.verbose=Verbose
+adafruit_feather_esp32s3.menu.PartitionScheme.noota_ffat.build.partitions=noota_ffat
+adafruit_feather_esp32s3.menu.PartitionScheme.noota_ffat.upload.maximum_size=2097152
+adafruit_feather_esp32s3.menu.PartitionScheme.noota_3gffat=No OTA (1MB APP/3MB FATFS)
+adafruit_feather_esp32s3.menu.PartitionScheme.noota_3gffat.build.partitions=noota_3gffat
+adafruit_feather_esp32s3.menu.PartitionScheme.noota_3gffat.upload.maximum_size=1048576
+adafruit_feather_esp32s3.menu.PartitionScheme.huge_app=Huge APP (3MB No OTA/1MB SPIFFS)
+adafruit_feather_esp32s3.menu.PartitionScheme.huge_app.build.partitions=huge_app
+adafruit_feather_esp32s3.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+adafruit_feather_esp32s3.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+adafruit_feather_esp32s3.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+adafruit_feather_esp32s3.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+adafruit_feather_esp32s3.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FAT)
+adafruit_feather_esp32s3.menu.PartitionScheme.fatflash.build.partitions=ffat
+adafruit_feather_esp32s3.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+adafruit_feather_esp32s3.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9MB FATFS)
+adafruit_feather_esp32s3.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+adafruit_feather_esp32s3.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+
+adafruit_feather_esp32s3.menu.CPUFreq.240=240MHz (WiFi)
+adafruit_feather_esp32s3.menu.CPUFreq.240.build.f_cpu=240000000L
+adafruit_feather_esp32s3.menu.CPUFreq.160=160MHz (WiFi)
+adafruit_feather_esp32s3.menu.CPUFreq.160.build.f_cpu=160000000L
+adafruit_feather_esp32s3.menu.CPUFreq.80=80MHz (WiFi)
+adafruit_feather_esp32s3.menu.CPUFreq.80.build.f_cpu=80000000L
+adafruit_feather_esp32s3.menu.CPUFreq.40=40MHz
+adafruit_feather_esp32s3.menu.CPUFreq.40.build.f_cpu=40000000L
+adafruit_feather_esp32s3.menu.CPUFreq.20=20MHz
+adafruit_feather_esp32s3.menu.CPUFreq.20.build.f_cpu=20000000L
+adafruit_feather_esp32s3.menu.CPUFreq.10=10MHz
+adafruit_feather_esp32s3.menu.CPUFreq.10.build.f_cpu=10000000L
+
+adafruit_feather_esp32s3.menu.FlashMode.qio=QIO
+adafruit_feather_esp32s3.menu.FlashMode.qio.build.flash_mode=dio
+adafruit_feather_esp32s3.menu.FlashMode.qio.build.boot=qio
+adafruit_feather_esp32s3.menu.FlashMode.dio=DIO
+adafruit_feather_esp32s3.menu.FlashMode.dio.build.flash_mode=dio
+adafruit_feather_esp32s3.menu.FlashMode.dio.build.boot=dio
+adafruit_feather_esp32s3.menu.FlashMode.qout=QOUT
+adafruit_feather_esp32s3.menu.FlashMode.qout.build.flash_mode=dout
+adafruit_feather_esp32s3.menu.FlashMode.qout.build.boot=qout
+adafruit_feather_esp32s3.menu.FlashMode.dout=DOUT
+adafruit_feather_esp32s3.menu.FlashMode.dout.build.flash_mode=dout
+adafruit_feather_esp32s3.menu.FlashMode.dout.build.boot=dout
```

Comment:
not all flash modes are available for S3 (and there are some extra). Please see the latest ESP32S3 Dev Board definitions for Flash and PSRAM

Reply:
should i just stick to QIO for the N8 type?

Reply:
TBH all our S3 boards (with QSPI) run just fine on 80MHz QIO (and you can even try 120MHz QIO), so you really do not need anything else

Reply:
time ago there were some ESP32 boards that required DIO or one even DOUT and the only place where it can happen now is if someone is not using our modules, but is using chip+own flash. I think we might remove all unnecessary options from S2 and C3 as well, before boards are too many.

Reply:
thanks i did find C3 like DOUT i think. i will remove the selection option from these boards and stick with QIO